sunpy/map/sources/soho.py

```diff
@@ -114,10 +118,11 @@ def __init__(self, data, header, **kwargs):
         # If non-standard Keyword is present, correct it too, for compatibility.
         if 'date_obs' in self.meta:
             self.meta['date_obs'] = self.meta['date-obs']
-
-        self._name = self.instrument + " " + self.detector + " " + self.measurement
+        self.meta['wavelnth'] = np.nan
+        self.meta['waveunit'] = 'nm'
```

Comment:
I agree that this was a bit of silliness on my part. It is a white-light instrument so broad band. We have no good way to deal with that.